win32/src/threadpool_win32.c

```diff
@@ -433,8 +420,7 @@ int threadpool_timer_start(THREADPOOL_HANDLE threadpool, uint32_t start_delay_ms
             else
             {
                 /* Codes_SRS_THREADPOOL_WIN32_42_006: [ threadpool_timer_start shall call CreateThreadpoolTimer to schedule execution the callback while passing to it the on_timer_callback function and the newly created context. ]*/
-                PTP_TIMER tp_timer = CreateThreadpoolTimer(on_timer_callback, timer_temp, &threadpool->tp_environment);
-
+                PTP_TIMER tp_timer = CreateThreadpoolTimer(on_timer_callback, timer_temp, NULL);

```

Comment:
I took this PR up through zprc and when you use the CreateThreadpoolTimer with the created tp_environment it throws an EV with Invalid Parameter on Cleanup.  I was reading some of the docs and it was mentioning that it could use the default Environment and when I remove the created Environment, it passes.  I've been looking for an explanation as to why use a created environment instead of the default environment, without much clarity.

Reply:
Why use the non-default threadpool is:
- Set the min/max number of threads and actually use it as a pool you control
- Be able to close this threadpool, which will call the closecleanup group and it will block on pending timers

If you want to decouple a timer from the threadpool, why not create a separate threadpool with the default (NULL) execution engine)?